msal/src/main/java/com/microsoft/identity/client/PublicClientApplication.java

```diff
@@ -777,10 +949,19 @@ private String getRequestTenantId(@NonNull final PublicClientApplicationConfigur
         }
 
         return result;
+
+    }
+
+    protected void validateAcquireTokenParameters(AcquireTokenParameters parameters){
+        return;
```

Comment:
nit: TODO?

Reply:
Not sure.  This is someone elses add... will leave as is for now.